content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,331 @@ Download the Istio release which includes installation files, samples, and the
-to evaluate Istio's features for traffic routing,
-fault injection, rate limiting, etc. Then, explore the various
-[Istio tasks](/docs/tasks/) that interest you.
+_Other environments (e.g., IBM Cloud Private, etc.):_
 
-## Next evaluation steps
+{{< text bash >}}
+$ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
+{{< /text >}}
 
-After installing Istio and deploying an application,
-see the following topics to explore more Istio features:
+{{< /tab >}}
+
+{{< /tabset >}}
+
+1.  Set `GATEWAY_URL`:
+
+    {{< text bash >}}
+    $ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+    {{< /text >}}
+
+1.  Ensure an IP address and port were successfully assigned to the environment variable:
+
+    {{< text bash >}}
+    $ echo $GATEWAY_URL
+    192.168.99.100:32194
+    {{< /text >}}
+
+### Verify external access {#confirm}
+
+Confirm that the Bookinfo application is accessible from outside the cluster by
+navigating in your browser to `http://<your $GATEWAY_URL value>/productpage`.
+
+## View the dashboard {#dashboard}
+
+Istio has several optional dashboards installed by the `demo` installation. The
+Kiali dashboard helps you understand the structure of your service mesh by
+displaying the topology and indicates the health of your mesh.
+
+1.  Access the Kiali dashboard. The default user name is `admin` and default password is `admin`.
+
+    {{< text bash >}}
+    $ istioctl dashboard kiali
+    {{< /text >}}
+
+1.  In the left navigation menu, select _Graph_ and in the _Namespace_ drop down, select _default_.
+
+    The Kiali dashboard shows an overview of your mesh with the relationships
+    between the services in the `Bookinfo` sample application. It also provides
+    filters to visualize the traffic flow.
+
+    {{< image link="./kiali-example2.png" caption="Kiali Dashboard" >}}
+
+## Next steps
+
+Congratulations on completing the installation!
+
+These tasks are a great place for beginners to start using Istio features:
 
 - [Request routing](/docs/tasks/traffic-management/request-routing/)
 - [Fault injection](/docs/tasks/traffic-management/fault-injection/)
 - [Traffic shifting](/docs/tasks/traffic-management/traffic-shifting/)
 - [Querying metrics](/docs/tasks/observability/metrics/querying-metrics/)
 - [Visualizing metrics](/docs/tasks/observability/metrics/using-istio-dashboard/)
-- [Collecting logs](/docs/tasks/observability/logs/access-log/)
 - [Rate limiting](/docs/tasks/policy-enforcement/rate-limiting/)
-- [Ingress gateways](/docs/tasks/traffic-management/ingress/ingress-control/)
 - [Accessing external services](/docs/tasks/traffic-management/egress/egress-control/)
 - [Visualizing your mesh](/docs/tasks/observability/kiali/)
 
-## Prepare for production deployment
-
-Before you install and customize Istio for production use,
-see the following topics:
+Before you customize Istio to fit your platform and intended use,
```

Comment:
This content seems out of place in getting started. I wonder if it should be promoted to its own page.

Reply:
It feels like "what is Istio". Ok, you installed Istio, ready to promote to prod?

Reply:
Reworded this section to be more clear about what we're telling the user to do.